### PR TITLE
feat(dockerfile): migrate fetching keys to script too

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
Previously we were including the ssh parameters at build time. now, we change it so that it is injected via env vars at run time. 

## Solution
- fetch via env vars and output to file

## New env vars
```
- ENV_TYPE=DEV
- DEV_SSH_PUBLIC_KEY=<take_from_your_own_ssh_config>
- DEV_SSH_PRIVATE_KEY=<take_from_your_own_ssh_config>
```

### CODE REVIEW

File: Dockerfile.develop [hunk 1]
```diff
@@ -2,10 +2,8 @@ FROM node:18-alpine AS base
 WORKDIR /opt/isomercms-backend
 RUN mkdir /root/.ssh
 COPY . .
-COPY ./.ssh /root/.ssh
-RUN chmod 600 /root/.ssh/github.pub
-RUN chmod 600 /root/.ssh/github
 RUN apk update
+RUN apk add --no-cache bash
```
Comment (QiluXie) [inline Dockerfile.develop:6]: nit#1: we can consider combing all these `RUN` commands in a single command:
```
RUN apk update && \
    apk add --no-cache bash git openssh-client
```
nit#2: add a line to remove apk cache after the installation

**Review by QiluXie — COMMENTED** [review]
left nit and non-blocking comments

File: Dockerfile.develop (hunk 1 above)
Comment (seaerchin) [reply Dockerfile.develop:6]: sure, sounds god

**Review by QiluXie — APPROVED** [review]
lgtm

**Comment by seaerchin** [comment]
### Merge activity

* **Dec 21, 12:46 AM**: @@seaerchin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/isomerpages/isomercms-backend/1070).

**Pull request merged**